plugins/module_utils/foreman_helper.py

```diff
@@ -170,6 +171,22 @@ def _patch_sync_plan_api(self):
         if next((x for x in _sync_plan_remove_products['params'] if x['name'] == 'organization_id'), None) is None:
             _sync_plan_remove_products['params'].append(_organization_parameter)
 
+    def _patch_cv_filter_rule_api(self):
+        """This is a workaround for missing params of CV Filter Rule update controller in Katello.
+           See https://projects.theforeman.org/issues/30908
+        """
```

Comment:
Should we think about limiting those "patches" to a range of server versions?
With the new `_validate_supported_payload` we rely very much on them to match the actual api exactly.

Reply:
Probably yes. This very patch is safe, as it only copies things if they exist for `create`, but still a very good idea.